php-path-traversal/common_2.php

```
@@ -0,0 +1,184 @@
+    // Open the zip file with ZipArchive
+    $zip = new ZipArchive();
+    if ($zip->open($zipFile) !== true) {
+        header("HTTP/1.1 500 Internal Server Error");
+        die("Failed to open the zip file.");
+    }
+
+    // Extract the files
+    for ($i = 0; $i < $zip->numFiles; $i++) {
+        $entryName = $zip->getNameIndex($i);
+        $entryDir = dirname($entryName);
+        $extractPath = $zipDir . "/" . $entryName;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$extractPath'. (undefined)

(UnusedLocalVariable)